### ISSUE TITLE
`voiceover@0.1.0` NPM package missing necessary files

### ISSUE THREAD

**Issue by cmorten** [AccessLint/voiceover.js#1]
It appears that some necessary files are missing in the published NPM package resulting in `voiceover.js` not working when installed.

For example, see <https://unpkg.com/browse/voiceover@0.1.0/lib/> where can see that the `./lib/` is missing the required files:

- `Commands.js`
- `VoiceOver.js`

Example error while trying to run tests for `auto-vo` locally also highlighting the issue.

```console
$ yarn test
yarn run v1.13.0
$ yarn build && mocha
$ tsc

Error: Cannot find module './Commands'
Require stack:
- /Users/craigmorten/git/guidepup/auto-vo/node_modules/voiceover/lib/index.js
- /Users/craigmorten/git/guidepup/auto-vo/lib/auto-vo.js
- /Users/craigmorten/git/guidepup/auto-vo/lib/index.js
```

I suspect something has gone awry in the last publish attempt(s).

It _could_ be that the `.gitignore` is interfering with the publish, as that is used by `npm publish` in the absense of a `.npmignore` - but speculating on how may be publishing the package!

Ref: https://docs.npmjs.com/cli/v8/using-npm/developers#keeping-files-out-of-your-package

**Comment by ckundo** [comment]
fixed in `v1.6.0`. thanks!
